src/main/scala/cromwell/engine/EngineFunctions.scala

```diff
@@ -1,18 +1,22 @@
 package cromwell.engine
 
 import cromwell.binding.WdlFunctions
-import cromwell.binding.values.{WdlInteger, WdlString, WdlValue}
+import cromwell.binding.values.{WdlFile, WdlInteger, WdlString, WdlValue}
 
 import scala.util.Try
 
 trait EngineFunctions extends WdlFunctions {
   protected def read_int(params: Seq[Try[WdlValue]]): Try[WdlInteger]
   protected def read_string(params: Seq[Try[WdlValue]]): Try[WdlString]
+  protected def stdout(params: Seq[Try[WdlValue]]): Try[WdlFile]
+  protected def stderr(params: Seq[Try[WdlValue]]): Try[WdlFile]
 
   def getFunction(name: String): WdlFunction = {
     name match {
       case "read_int" => read_int
       case "read_string" => read_string
+      case "stdout" => stdout
```

Comment:
The more I think about this the more my preference would be to keep the physical mapping and switch to scala-style internally. The interface is self-documenting and what we call things internally don't need to map (even word-for-word, much less formatting) w/ the WDL.

If the outer functions *are* kept in WDL naming, then I definitely would want it to be reflectively matched.